inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -87,9 +89,12 @@ def fuzzy_match(obj, eng):
     """
     if not current_app.config.get('FEATURE_FLAG_ENABLE_FUZZY_MATCHER'):
         return False
-
+    math_ml_latex_regex = '<math[^>]*>|<\/math> | (?<!\\)\$.*?(?<!\\)\$|(?<!\\)\\\(.*?(?<!\\)\\\)'
```

Comment:
this is wrong too, you don't want to strip only math tags, take a look at the example I gave you previously (and use that one as a test case)! Also, you didn't include `\[...\]`in the LaTeX pattern.